ampeg/src/test/java/HomePage.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage extends PageObject {
+	public final String uRL = "index.html";
+	
+	public HomePage(WebDriver driver, String baseURL) {
+		super(driver, baseURL);
+		setURL(uRL);
+	}
+	
+	public HomePage navigateDirect() {
+		navigateDirectURL();
+		
+		return this;
+	}
+	
+	public HomePage clickSearch() {
+		var searchLink = this.driver.findElement(By.xpath("//a[@title='Search']"));
+		
+		searchLink.click();
+		
+		return this;
+	}
+	
+	public HomePage sendSearchBox(String query) {
```

Comment:
sendSearchBox doesn't really describe what the method does. Rename to something like 'search' or 'enterSearchText'